tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -1049,4 +1050,37 @@ abstract class NodeJsActionContainerTests extends BasicActionRunnerTests with Ws
         e shouldBe empty
     })
   }
+
+  Map(
+    "prelaunched" -> Map.empty[String, String],
+    "non-prelaunched" -> Map("OW_INIT_IN_ACTIONLOOP" -> "")
+  ).foreach { case (name, env) =>
+    it should s"support a function with Lambda signature exported via CommonJS ($name)" in {
+      val (out, err) = withActionContainer(env) { c =>
+        val code =
+          """
+            | module.exports.main = async function (event, context) {
+            |     return {
+            |       event,
+            |       contextKeys: Object.keys(context),
+            |     };
+            | }
+          """.stripMargin
+
+        val (initCode, _) = c.init(initPayload(code))
+        initCode should be(200)
+
+        val (runCode, out) = c.run(runPayload(
+          JsObject(
+            "__ow_headers" -> "x".toJson,
+            "__ow_method" -> "get".toJson,
+            "__ow_path" -> "y".toJson)
```

Comment:
In the followup PR to this, we should pick up checking the entire context's fields (including `getRemainingTimeInMillis`) as is done in https://github.com/nimbella/openwhisk-runtime-python/pull/11. I'm fine to relay that to the next PR though.

Reply:
Will save this for next PR.